core/src/main/java/org/apache/carbondata/core/datastore/chunk/impl/DimensionRawColumnChunk.java

```diff
@@ -197,6 +197,10 @@ public static CarbonDictionary getDictionary(LocalDictionaryChunk localDictionar
           localDictionaryChunk.getDictionary_data().length);
       BitSet usedDictionary = BitSet.valueOf(compressor.unCompressByte(
           localDictionaryChunk.getDictionary_values()));
+      // if dictionary is empty, return null
+      if (usedDictionary.isEmpty()) {
```

Comment:
DataFile class call this method，but it use the return value directly, it will throw NullException. i think it need be protected
Please see DataFile line 360,
CarbonDictionary dictionary = DimensionRawColumnChunk.getDictionary(dictionaryChunk, comp);
blockletDictionaryEntries = dictionary.getDictionaryActualSize();